moqt-server/src/lib.rs

```diff
@@ -51,7 +51,7 @@ use wtransport::{
     endpoint::IncomingSession, Endpoint, Identity, RecvStream, SendStream, ServerConfig,
 };
 type SubscribeId = u64;
-type SenderToOpenSubscription = Sender<(SubscribeId, DataStreamType)>;
+pub(crate) type SenderToOpenSubscription = Sender<(SubscribeId, DataStreamType)>;
```

Comment:
Why did you use not `pub type` but  `pub(crate)`?

Reply:
Because I'd like to minimize the range.